### PR TITLE
Parser for Hitachi TIFF, fixes #44

### LINKED ISSUE #44: Add parsing for Hitachi

### PR DESCRIPTION
**Pull request by mkuehbach**

### CODE REVIEW

File: src/pynxtools_em/utils/tfs_utils.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+from typing import List
+
+from pynxtools_em.configurations.image_tiff_tfs_cfg import (
+    TIFF_TFS_ALL_CONCEPTS,
+    TIFF_TFS_PARENT_CONCEPTS,
+)
+
+
+def get_fei_parent_concepts() -> List:
```
Comment (lukaspie) [inline src/pynxtools_em/utils/tfs_utils.py:28]: Is this strictly necessary. Why not import and use `TIFF_TFS_PARENT_CONCEPTS` directly?

File: src/pynxtools_em/parsers/image_tiff_hitachi.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            self.supported = True
+
+    def parse(self, template: dict) -> dict:
+        if self.supported is True:
+            print(f"Parsing via Hitachi...")
+            # metadata have at this point already been collected into an fd.FlatDict
+            self.process_event_data_em_metadata(template)
+            self.process_event_data_em_data(template)
+        else:
+            print(
```
Comment (lukaspie) [inline src/pynxtools_em/parsers/image_tiff_hitachi.py:129]: logger.warning?

File: src/pynxtools_em/utils/tfs_utils.py (hunk 1 above)
Comment (mkuehbach) [reply src/pynxtools_em/utils/tfs_utils.py:28]: Good catch!, its not, changed it

File: src/pynxtools_em/parsers/image_tiff_hitachi.py (hunk 2 above)
Comment (mkuehbach) [reply src/pynxtools_em/parsers/image_tiff_hitachi.py:129]: It is correct, I am still using print some maybe not even few places where logging would be better, agreed, but for the future, opened as a separate issue, as this is also cross-parser relevant, #62.

**Pull request merged**